### DISCUSSION TITLE
Questions about the application process for the GSoC 2021

### DISCUSSION THREAD

**Discussion by JosueDavalos** [alan-turing-institute/sktime#742] [General]
Hi, my name is Josue Davalos, a CS student in Ecuador. This will be my first time working on an open-source project and I would like to work with you. I love the artificial intelligence area and I have worked analyzing time series for almost 2 years (analyzing malware, detecting anomalies, predictions, etc).

I have a question about the application, I have read that to apply to this program applicants must submit a proposal. In the section you wrote about how to apply and the selection process, you do not mention the proposal, you mention about filling out a form until April 13 (the time where GSoC applications end).

How should I structure my proposal?
Do you have a format for this proposal?

Thank you very much for your attention.
Josue Davalos

**Comment by fkiraly** [comment]
Have a look here at the application process:
https://github.com/alan-turing-institute/sktime/issues/731

The "proposal" would be a proposal what to work on, e.g., a pick of topic, or a suggested feature extension.
You can base this on the list of suggested projects - or your own independent ideas, which are also very much appreciated.

**Reply by mloning** [reply]
For a rough template, take a look at the template for our [enhancement proposal](https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md).

**Comment by JosueDavalos** [comment]
I see, thank you very much @mloning @fkiraly 

I would like to work on  "Time series clustering" project, I have worked with time series even I have tried to address this problem in the production of shrimp pools. I also find project "Forecasting" very attractive. I would like very much to contribute with you

One last question, where do I find the entrance tasks you mention?

**Reply by mloning** [reply]
The entrance task can be any contribution to sktime via a pull request, check out the linked contributing guide to find out how to get started. The tutorials are also a good place to get some ideas on what might be missing or needs improvement.

**Comment by mloning** [comment]
@JosueDavalos we had another look at our application process and now included a step for the submission of the proposal. You can find the up-to-date version here: https://github.com/alan-turing-institute/sktime/wiki/GSoC-2021

Thanks for pointing out that this wasn't clear and sorry for any confusion this caused.

**Reply by fkiraly** [reply]
yes, sorry for not being clearer in my original post about this.
This is an extra GSoC-facing step.

**Reply by JosueDavalos** [reply]
okay, Thanks so much for the informacion.

**Reply by JosueDavalos** [reply]
@mloning @fkiraly I have finished my draft of the proposal., and I would like to know if you think you can provide me with examples of proposals accepted at GSoC in the past years? or where can I find it?

**Reply by mloning** [reply]
This is our first time participating, there's a coffee chat tomorrow in case you want to discuss some questions (see Gitter/Discord)

**Reply by JosueDavalos** [reply]
I see, thank you
